Task_1/Task_1/src/components/to_do_list/List.jsx

```diff
@@ -0,0 +1,23 @@
+import TaskBar from "./task_bar";
+
+const List = (props) => {
+  if (props.list.length == 0) {
+    return (
+      <div className="mt-100px w-full text-center mt-[250px] mb-[200px]">
+        <h1 className="text-5xl font-bold  mt-100px mb-[100px]">To Do List</h1>
+        <h1 className="text-2xl font-bold  mt-100px mb-[100px] w-[600px] mx-auto h-[4   0px] rounded-2xl bg-indigo-300">List is Empty</h1>
+      </div>
+    );
+  }
+
+  return (
+    <div className="mt-100px w-full text-center mt-[250px] mb-[200px]">
+      <h1 className="text-5xl font-bold  mt-100px mb-[100px]">To Do List</h1>
+      {props.list.map((obj) => (
```

Comment:
what if props.list is undefined ? it will break application. use optional chaining here. @codal-uchoudhary

Reply:
@codal-shubham , here i am list passing from app component to the  List component. i added logic for empty list .

Reply:
Updated using optional chaining